src/views/Space/Overview.vue

```diff
@@ -52,6 +54,10 @@ const grouped = computed(() => {
     return v;
   }, initialValue);
 });
+
+watchEffect(() => {
+  setTitle(`Overview - ${props.space.name}`);
```

Comment:
For overview page we can use the space name
```suggestion
  setTitle(props.space.name);
```